setup.cfg

```diff
@@ -36,25 +36,12 @@ galaxy =
     templates/*.html
     templates/**/*.html
 
-[tool:pytest]
```

Comment:
Why these settings were moved outside configuration file?

Reply:
UPD: Moving coverage options seems reasonable to me, but why to move `DJANGO_SETTINGS_MODULE`?

Reply:
I moved this to facilitate running subsets of the full test suite, which is essential for swift workflows and also for `pytest` flags like `--ll` and `--lf`. In these cases the coverage generated is totally invalid, having been generated by a test run that didn't include all the relevant tests. So, I moved the flags from the default configuration to the invocation, in the Makefile, so they could differ based on use.

Reply:
Ok, thanks for explanation.